src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -228,7 +351,7 @@ else if(stepFieldClass == Float.class)
                 else if(stepFieldClass == Integer.class)
                     mapping.put(entry.getKey(), (Integer) entry.getValue());
                 else
-                    logger.log(Level.WARNING, stepFieldClass.getName() + "is not supported at this time.");
+                    logger.log(Level.WARNING, stepFieldClass.getName() + " is not supported at this time.");
```

Comment:
This should be an error too.

Reply:
If some parameter types are not supported then we can ignore them and create a step using supported parameter types.

Reply:
For the alpha release it is okay to issue warnings, but warnings easily cause silent failures so for the final release I think this needs to be errors. Please create a Jira to remind us to review all warnings.